packaging/Makefile

```diff
@@ -1,64 +1,60 @@
-.PHONY: docker-push
-
 VERSION            ?= $(shell git rev-parse HEAD)
 REGISTRY           ?= registry-write.opensource.zalan.do/teapot
 BINARIES           ?= skipper webhook eskip routesrv
 IMAGE              ?= $(REGISTRY)/skipper:$(VERSION)
 ARM64_IMAGE        ?= $(REGISTRY)/skipper-arm64:$(VERSION)
 ARM_IMAGE          ?= $(REGISTRY)/skipper-armv7:$(VERSION)
-PACKAGE            ?= github.com/zalando/skipper
+MULTIARCH_IMAGE    ?= container-registry-test.zalando.net/teapot/skipper:$(VERSION)
 CGO_ENABLED        ?= 0
-GOOS               ?= linux
-GOARCH             ?= amd64
-GOARM              ?=
-GO111              ?= on
 COMMIT_HASH        = $(shell git rev-parse --short HEAD)
 
-default: docker-build
-
-skipper:
-	GO111MODULE=$(GO111) \
-	GOOS=$(GOOS) \
-	GOARCH=$(GOARCH) \
-	$(GOARM) \
-	CGO_ENABLED=$(CGO_ENABLED) \
-	go build -ldflags "-X main.version=$(VERSION) -X main.commit=$(COMMIT_HASH)" -o skipper ../cmd/skipper/*.go
+.PHONY: help
+help: ## display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-.]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
-eskip:
-	GO111MODULE=$(GO111) GOOS=$(GOOS) GOARCH=$(GOARCH) $(GOARM) CGO_ENABLED=$(CGO_ENABLED) go build -o eskip ../cmd/eskip/*.go
+.PHONY: clean
+clean: ## clean binaries
+	rm -rf sha256sum.txt *.tar.gz build/
 
-webhook:
-	GO111MODULE=$(GO111) GOOS=$(GOOS) GOARCH=$(GOARCH) $(GOARM) CGO_ENABLED=$(CGO_ENABLED) go build -o webhook ../cmd/webhook/*.go
+.PHONY: docker.build.local
+docker.build.local: clean $(addprefix build/,$(BINARIES)) ## build local architecture image
+	docker build -t $(IMAGE) --build-arg TARGETPLATFORM= .
 
-routesrv:
-	GO111MODULE=$(GO111) GOOS=$(GOOS) GOARCH=$(GOARCH) $(GOARM) CGO_ENABLED=$(CGO_ENABLED) go build -o routesrv ../cmd/routesrv/*.go
+.PHONY: docker.build.amd64
+docker.build.amd64: clean build.linux.amd64 docker.build.enable ## build linux/amd64 image using a trusted amd64 alpine base image
+	docker buildx build -t $(IMAGE) --platform linux/amd64 --load . \
+		--build-arg BASE_IMAGE=registry.opensource.zalan.do/library/alpine-3:latest
```

Comment:
Why use a custom base image here? Is it for compliance with Zalando? The default `alpine:latest` would work also work.

Reply:
`container-registry.zalando.net/library/alpine-3:latest` would also work as it's just the compliant version of `alpine:latest` (more correctly `alpine:3`).

Reply:
> Is it for compliance with Zalando? 

Yes, so the idea is that Docker file uses opensource image and Makefile configured base image we need for us

Reply:
Then best use `alpine:latest` (or with a version) by default for open source and overwrite it in Makefile with `container-registry.zalando.net/library/alpine-3:latest` (for us).

Reply:
Latest will break OpenSSF Scorecard because they want sha based image versions. I think it's also better being explicit of versions and have automations that change it. In the future we can likely merge these pr automatically as discussed in an internal meeting.